From: Antonio Radici <antonio@debian.org>
Updated-By: наб <nabijaczleweli@nabijaczleweli.xyz>
Date: Thu, 27 Feb 2014 16:06:15 +0100
Updated-Date: Sat, 06 Nov 2021 14:13:41 +0100
Subject: use_usr_bin_editor

Even if EDITOR is not set, mutt will always use /usr/bin/editor
(which is set by update-alternatives), rather than falling back
to vi.
---
 init.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/init.c b/init.c
index fb47517..d4d3855 100644
--- a/init.c
+++ b/init.c
@@ -432,7 +432,7 @@ int mutt_init(struct ConfigSet *cs, const char *dlevel, const char *dfile,
   if (!env_ed)
     env_ed = mutt_str_getenv("EDITOR");
   if (!env_ed)
-    env_ed = "vi";
+    env_ed = "/usr/bin/editor";
   cs_str_initial_set(cs, "editor", env_ed, NULL);
 
   const char *const c_editor = cs_subset_string(NeoMutt->sub, "editor");
